src/ensembl/production/metadata/updater/core.py

```diff
@@ -542,7 +543,22 @@ def new_genebuild(self, species_id, meta_session, db_uri, source=None):
             attribute=sample_location_attribute,
         )
         genebuild_dataset_attributes.append(sample_location_param)
-
+        # Add the production name:
+        production_name_attribute = meta_session.query(Attribute).filter(
+            Attribute.name == "production.production_name").one_or_none()
+        if production_name_attribute is None:
```

Comment:
Personally, I would advise not to automatically create this one, but just raise an error if the attribute does not exists.